widget/view.controller.js

```diff
@@ -0,0 +1,89 @@
+/* Copyright start
+  Copyright (C) 2008 - 2023 Fortinet Inc.
+  All rights reserved.
+  FORTINET CONFIDENTIAL & FORTINET PROPRIETARY SOURCE CODE
+  Copyright end */
+  'use strict';
+  (function () {
+    angular
+      .module('cybersponse')
+      .controller('vtAugment100Ctrl', vtAugment100Ctrl);
+    vtAugment100Ctrl.$inject = ['$scope', '$sce', '$resource', 'API', '_', 'toaster', '$rootScope', '$state', 'Entity'];
+  
+    function vtAugment100Ctrl($scope, $sce, $resource, API, _, toaster, $rootScope, $state, Entity) {
+      $scope.renderingURL = { url: '' }
+      $scope.loadProcessing = true;
+      $scope.healthCheck = true;
+      $scope.connectorConfig = true;
+      var vtConfig = null;
+      $scope.module = $state.params.module; //Get module name
+      $scope.moduleUUID = $state.params.id; //Get module uuid
+      var entity = new Entity($scope.module); //Create module entity
+      entity.get($scope.moduleUUID, {
+          $relationships: true
+          }).then(function(){
+             $scope.indicatorValue = entity.fields[$scope.config.rows[0].columns[0].fields[0].name].value;
+      }); //Get module fields values
+      
+      
+      function _getConfigConnectorList() {
+        $resource(API.INTEGRATIONS + 'connector_details/?format=json&configured=true').save({
+          'page_size': 300
+        }, function (data) {
+          vtConfig = _.find(data.data, function (item) { return item.label === 'VirusTotal Premium';});
+          if (typeof vtConfig == 'undefined'){
```

Comment:
use angular.isUndefined(vtConfig) instead of typeof

Reply:
done